base/applications/msconfig_new/systempage.cpp

```diff
@@ -728,9 +728,9 @@ CommonWndProc(HWND hDlg, UINT message, WPARAM wParam, LPARAM lParam)
                          * to the message.
                          *
                          * If the dialog box procedure processes a message that requires a specific return value, the dialog box procedure
-                         * should set the desired return value by calling SetWindowLong(hwndDlg, DWL_MSGRESULT, lResult) immediately before
+                         * should set the desired return value by calling SetWindowLong(hwndDlg, DWLP_MSGRESULT, lResult) immediately before
```

Comment:
Can you fix the comment that mentions "SetWindowLong" instead of the Ptr version? thanks! (this is part of my code so I allow touching it).